pdf_tools.go

```diff
@@ -0,0 +1,180 @@
+package docconv
```

Comment:
It would be better if all OCR related funcs were moved to the same file and excluded with the build tags. This file has a mix of both